railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -33,20 +32,6 @@ def db_create_and_drop(namespace, expected_database)
         end
       end
 
-      def db_create_and_drop_namespace(namespace, expected_database)
-        Dir.chdir(app_path) do
-          output = rails("db:create:#{namespace}")
-          assert_match(/Created database/, output)
-          assert_match_namespace(namespace, output)
-          assert File.exist?(expected_database)
-
-          output = rails("db:drop:#{namespace}")
```

Comment:
Similar, but `rails("db:drop:#{namespace}")` is not the same with `rails("db:drop")`.